src/main/java/service/impl/ReaderServiceImpl.java

```diff
@@ -0,0 +1,21 @@
+package service.impl;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import service.ReaderService;
+
+public class ReaderServiceImpl implements ReaderService {
+    @Override
+    public List<String> readDataFromFile(String filePath) {
+        List<String> dataFromFile = new ArrayList<>();
+        try {
+            dataFromFile = Files.readAllLines(Path.of(filePath));
+        } catch (IOException e) {
+            throw new RuntimeException("Can`t read data from file" + filePath);
+        }
+        return dataFromFile;
```

Comment:
![image](https://github.com/mate-academy/jv-fruit-shop/assets/20326858/b7aa36b4-14c2-4052-88e8-cd650a38c5f1)

```suggestion
        try {
            return Files.readAllLines(Path.of(filePath));
        } catch (IOException e) {
            throw new RuntimeException("Can`t read data from file" + filePath, e);
        }
```